personio-personnel-data-api-oa3.yaml

```diff
@@ -2824,6 +3018,35 @@ components:
               example: "Project name"
             active:
               example: true
+              description: "Marks the availability of the project. When a new project is created, it's set by default to false."
```

Comment:
I would expect the second part of this description in the request description. You can simply state `The default value is false`